### PR TITLE
add test case to check to_consistent use

### PR DESCRIPTION
**Pull request by wyushun**
According to @daquexian comment in [change consistent name to global in python folder](https://github.com/Oneflow-Inc/oneflow/pull/7429#discussion_r807510777), make following changes:
 - change function signature to ` _to_consistent(self, *args, **kwargs)`
 - add related test case

reference: [How do you test that a Python function throws an exception](https://stackoverflow.com/questions/129507/how-do-you-test-that-a-python-function-throws-an-exception)

test command:
```
export ONEFLOW_TEST_DEVICE_NUM=2; python -m oneflow.distributed.launch --nproc_per_node 2 exceptions/test_to_consistent_error.py
```

### CODE REVIEW

**Comment by wyushun** [comment]
reference: https://stackoverflow.com/questions/129507/how-do-you-test-that-a-python-function-throws-an-exception/3166985#3166985

File: python/oneflow/framework/tensor.py [hunk 1]
```diff
@@ -971,7 +971,7 @@ def _is_consistent(self):
     raise RuntimeError(".is_consistent has been removed, please use .is_global instead")
 
 
-def _to_consistent(self):
+def _to_consistent(self, *args, **kwargs):
```
Comment (daquexian) [inline python/oneflow/framework/tensor.py:974]: 可能还有 module 的 to_consistent 也需要修改

File: python/oneflow/test/exceptions/test_to_consistent_error.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    def test_to_consistent(test_case):
+        try:
+            data = flow.rand(2, dtype=flow.float32)
+            placement = flow.env.all_device_placement("cuda")
+            sbp = flow.sbp.split(0)
+            global_data = data.to_consistent(placement=placement, sbp=sbp)
+
+        except Exception as e:
+            err_msg = ".to_consistent has been removed, please use .to_global instead"
+            assert err_msg in str(e)
```
Comment (daquexian) [inline python/oneflow/test/exceptions/test_to_consistent_error.py:42]: 可以用 unittest 库的内置功能而不手动 try-except（目前的手动 try-except 就没有检查到 “一定发生了 exception” 这一点）：https://stackoverflow.com/questions/129507/how-do-you-test-that-a-python-function-throws-an-exception/3166985#3166985

File: python/oneflow/framework/tensor.py (hunk 1 above)
Comment (wyushun) [reply python/oneflow/framework/tensor.py:974]: module里的定义是这样的：
```
    def to_consistent(self, placement=None, sbp=None):
        raise RuntimeError(".to_consistent has been removed, please use .to_global instead")
```
需要修改吗

File: python/oneflow/test/exceptions/test_to_consistent_error.py (hunk 2 above)
Comment (wyushun) [reply python/oneflow/test/exceptions/test_to_consistent_error.py:42]: ok，我看下

Comment (wyushun) [reply python/oneflow/test/exceptions/test_to_consistent_error.py]: update.

File: python/oneflow/framework/tensor.py (hunk 1 above)
Comment (wyushun) [reply python/oneflow/framework/tensor.py:974]: updated

File: python/oneflow/test/exceptions/test_to_consistent_error.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+import time
+import oneflow as flow
+import oneflow.unittest
+
+from oneflow.test_utils.automated_test_util import *
+
+
+@flow.unittest.skip_unless_1n2d()
+class TestToConsistentError(flow.unittest.TestCase):
+    @autotest(n=1, check_graph=True)
```
Comment (daquexian) [inline python/oneflow/test/exceptions/test_to_consistent_error.py:31]: 这个不用加 autotest（目前实际应该也没有使用），以及可以把 is_consistent 和 module.to_consistent 也覆盖在测试里

Comment (wyushun) [reply python/oneflow/test/exceptions/test_to_consistent_error.py]: updated.

**Review by daquexian — APPROVED** [review]

**Review by clackhan — APPROVED** [review]

**Pull request merged**